test/unit/test-error.js

```diff
@@ -585,11 +592,13 @@ describe('getErrorReportData', () => {
   it('should throttle user errors', () => {
     nextRandomNumber = 0.2;
     let e = '';
-    try {
-      userAssert(false, 'XYZ');
-    } catch (error) {
-      e = error;
-    }
+    allowConsoleError(() => {
```

Comment:
I wonder if we could somehow automatically apply `allowConsoleError` for all error logs within `userAssert`

Reply:
Not a bad thought. Might be accomplished via stubbing when the sandbox is setup, but would need to be compatible with current tests that explicitly call `allowConsoleError()`. It could be a convenient means of squashing sinon warnings in extensions (for which I have no plans of applying these manual fixes).